app/org/sagebionetworks/bridge/services/NotificationsService.java

```diff
@@ -106,6 +128,45 @@ public void deleteRegistration(String healthCode, String guid) {
         
         notificationRegistrationDao.deleteRegistration(healthCode, guid);
     }
+    
+    /**
+     * Send a push notification to an individual participant (assuming they have registered for push notification). 
+     * This mechanism is intended to message specific individuals, <i>and should not be used to send out notifications 
+     * to many accounts.</i> Create a topic, ask your users to subscribe to that topic in your application, and message 
+     * them via that topic.
+     */
+    public void sendNotification(StudyIdentifier studyId, String healthCode, NotificationMessage message) {
+        checkNotNull(studyId);
+        checkNotNull(healthCode);
+        checkNotNull(message);
+        
+        Validate.entityThrowingException(NotificationMessageValidator.INSTANCE, message);
+        
+        List<NotificationRegistration> registrations = notificationRegistrationDao.listRegistrations(healthCode);
+        if (registrations.isEmpty()) {
+            throw new BadRequestException("Participant has not registered to receive push notifications.");
```

Comment:
From what I understand, it's expected for participants not to receive notifications since they can opt out. Would it be better not to throw an exception since non-delivery is an expected use case, maybe return a 204? Would it be useful to provide info in the response along the lines of how many devices it was delivered to, which devices, etc?